elm-package.json

```diff
@@ -10,7 +10,8 @@
         "Test",
         "Test.Runner",
         "Expect",
-        "Fuzz"
+        "Fuzz",
+        "Float"
```

Comment:
@mgold it is exposed, because I wanted the documentation in html format. I'll just revert that change again.